# We need to talk about Polygenic Risk Scores 
## Dr Lynsey S Hall
_Written for publication in the [Genetics Society Newsletter Issue 81, July 2019](https://genetics.org.uk/wp-content/uploads/2019/09/Issue-81-v3.pdf)_

Polygenic risk scores (or as I prefer to call them, polygenic profile scores, to avoid the notion of being at risk of non-disease traits such as height) were one of the first analyses I learned how to perform when I haphazardly entered the world of statistical genetics in 2012. Part of the beauty of these scores is their simplicity. Even as someone who, at that time, had no prior computing experience, a limited grasp of statistics, and a pretty rudimentary understanding of genetics, I could follow the online tutorial and generate these scores - a very satisfying feeling for a rather overwhelmed 1st year PhD student. At that time, my understanding of these scores was, understandably, very superficial and limited to their creation rather than their limitations. One simply multiplied the number of copies of a genetic variant in an individual by its evidence for association with the trait of interest, repeat and sum across the genome, and hey presto - polygenic profile scores! Available for birthdays, bar mitzvahs and all your cross-trait associative needs! 

The idea behind polygenic profile scores (PGS) is to estimate an individual’s genetic propensity for a trait, attributable to common genetic variants, with two main applications in mind: prediction and association. The seminal PGS paper, published in 2009 by Purcell et al [1], took the animal genetics concept of the genetic breeding value (a measure of an individual’s aggregate additive genetic load for a trait, of which approximately 50% is passed on to their offspring), and applied it in a human genetics context by using trait-associated alleles from one cohort to test for association in independent cohorts. They demonstrated that schizophrenia cases had a higher mean PGS compared to controls in the replication dataset (indicating that scoring alleles identified in the discovery GWAS were significantly enriched among independently ascertained cases), and that schizophrenia PGS showed an association with bipolar disorder case/control status, but not six clinically distinct non-psychiatric traits, suggesting a shared genetic component between schizophrenia and bipolar disorder. Since this paper, the PGS literature has exploded, with the general trend being that PGS are pretty good for testing cross-trait association but are (currently) pretty pants when it comes to prediction. And therein the problem lies.  

Fast-forward 9 years from that paper, and I open Twitter to see an article in New Scientist [2] outlining how a company, Genomic Prediction, had “developed genetic screening tests that can assess complex traits, such as the risk of some diseases and low intelligence, in IVF embryos”. In other words, generating PGS in embryos to aid selection. My immediate reaction upon reading this was something to the tune of “HOLD THE PHONE, THAT DOES NOT SEEM LIKE A GOOD IDEA.” My objections are several fold, which I cover below. 

Firstly, put simply, PGS aren’t currently predictively accurate, and are unlikely to become predictively accurate in the near future either, unless sample sizes increase by multiple orders of magnitude. Using intelligence as an example (bearing in mind this is one of the traits Genomic Prediction are offering their services for), the largest study to date estimates the variation in intelligence explained by PGS at 5.2% [3]. Meaning 94.8% of the variation we observe in intelligence is not being captured by these scores. This is impressively low for a sample size of 269867 individuals, especially given the maximum variance explained by genetic factors (i.e. the heritability) is estimated to be between 50-80% [4].  

Furthermore, heritability refers to a population-level estimate - not how much of a trait is “caused by” genes, which is a common misconception. Therefore, taking several embryos created by two individuals isn’t actually going to result in a great degree of divergent variance to choose from. Genomic breeding values work in an animal breeding context because we are interested in the metric at the group, and not the individual, level and because we can artificially drive selection in an animal population. This results in a more closely related group, which results in less recombination, affording improved predictive accuracy in a way which is much less tractable in an effectively randomly mating outbred population such as the human race [5]. 

Heritability estimates, the upper bound of PGS predictive ability, are specific to ancestral populations. This brings me onto another glaring issue regarding the clinical utility of PGS: the Eurocentric bias of genetic studies. As different ancestral populations have different patterns of linkage disequilibrium, resulting in different frequencies of both tag SNPs and casual variants - studying mixed populations can result in spurious associations (in particular, if cases are recruited from one population, and controls from another). To try and combat this, genetic studies have generally used samples that are as genetically homogeneous as possible, with the majority of these studies focussing on white European ancestry. 

An example of this bias can be seen in work by Levy-Sakin et al [6] who looked at genomes from 26 ethnic populations and identified 60Mb of non-redundant genome content that was not present in the reference genome (which geneticists constructed as a “representative example” of the human genome). Another paper by Sherman et al [7] showed that in 910 AfricanAmerican genomes, 10% of the genome was present in all the study individuals but, again, missing from the reference genome. 

The weights used in PGS are taken from genome-wide association studies (GWAS) of the trait of interest, which more often than not have been conducted in a population of European ancestry. Indeed, according to the GWAS catalogue, ~79% of GWAS participants are of European ancestry, despite this subpopulation only comprising ~16% of the global population. This is problematic when generating PGS in non-European ancestry individuals, as PGS predict individual risk far more accurately in Europeans than non-Europeans; to the extent that in individuals of African descent, the predictive power of PGS derived from European data is no better than chance [8]. 

So, the story so far - we have a company offering to aid embryo selection using PGS, we know that the trait variance explained by PGS in one of the traits on offer (intelligence) is only about 5% and, even then, that value is only valid in about 16% of the global population. Ideal. Money well spent. But, even in a world where the variance explained by PGS matched the trait heritability estimates from twin and family studies (which is implausible given the likely role of rare and nonadditive genetic effects - inherently not captured by PGS), and we had studies on all different ancestries - selecting embryos based on PGS still seems a fairly ropey idea to me, and that is due to pleiotropy. 

Pleiotropy is where genetic variants influence more than one trait. Indeed, it is the basis of the whole non-predictive side of PGS analysis - association. PGS analyses have shown repeatedly that many pairs of traits appear to share a common genetic basis. Therefore, by selecting on one trait, we may inadvertently be selecting on another, perhaps less desirable, trait. Given how little we know about the genetic architecture of complex diseases, it would seem folly to start selecting embryos based on their PGS for one trait. Admittedly, Genomic Prediction are currently only offering the service to rule out embryos who are more than 25 standard deviations below the population mean but have not ruled out selecting on higher than average IQ in the future, citing the rationale of “if they don’t, someone else will”.  

The story for PGS is not all doom and gloom, however. As stated above, they are useful at indicating potential shared biology between pairs of traits. And, in the world of precision medicine, have been shown to predict breast cancer, prostate cancer and type 1 diabetes in European descent patients more accurately than current clinical models [8]. So, the above is not to undermine the clinical potential of PGS. Just, with so much further research being required (in many instances, in data that is yet to be collected), maybe spend your hard-earned cash on something more conducive to your happiness than trying to predict which embryo will be the smartest. 

[1] Purcell et al (2009, Nature) Common polygenic variation contributes to risk of schizophrenia and bipolar disorder 

[2] New Scientist (2018) A new test can predict IVF embryos’ risk of having a low IQ 

[3] Savage et al (2018, Nature Genetics) Genome-wide association meta-analysis in 269,867 individuals identifies new genetic and functional links to intelligence 

[4] Posthuma et al (2001, Behavior Genetics) Perceptual Speed and IQ Are Associated Through Common Genetic Factors 

[5] Wray et al (2019, Genetics) Complex Trait Prediction from Genome Data: Contrasting EBV in Livestock to PRS in Humans 

[6] Levy-Sakin et al (2019, Nature Communications) Genome maps across 26 human populations reveal population-specific patterns of structural variation 

[7] Sherman et al (2019, Nature Genetics) Assembly of a pangenome from deep sequencing of 910 humans of African descent 

[8] Martin et al (2019, Nature Genetics) Clinical use of current polygenic risk scores may exacerbate health disparities
